src/app/content/search/components/SearchResultsSidebar/SearchResultHits.tsx

```diff
@@ -65,47 +65,65 @@ const OneSearchResultHit = ({
   onClick,
   selectedResult,
   queryParams,
-  loader,
-}: OneSearchResultHitProps) => {
-  const pair = useKeyTermPair({ hit, loader });
-  // tslint:disable-next-line: variable-name
-  const HighlightPreview = React.useCallback(
-    (highlight: string, index: number) => {
-      const thisResult = { result: hit, highlight: index };
-      const isSelected = isEqual(selectedResult, thisResult);
-      const target: SearchScrollTarget = {
-        elementId: thisResult.result.source.elementId,
-        index,
-        type: 'search',
-      };
+  index,
+  highlight,
+}: OneSearchResultHitProps & {
+  index: number;
+  highlight: string;
+}) {
+  const thisResult = { result: hit, highlight: index };
+  const isSelected = isEqual(selectedResult, thisResult);
+  const target: SearchScrollTarget = {
+    elementId: thisResult.result.source.elementId,
+    index,
+    type: 'search',
+  };
+  const divRef = React.useRef<HTMLDivElement>(null);
+  const excerptText = divRef.current?.textContent;
+  const page = getPage(hit);
+  const ariaLabel = `${excerptText} in ${page.slug}`;
```

Comment:
do we also need to do like `result number ${index+1} in ${page.slug}: ${excerptText}` or do you think this is unique enough?

Reply:
We'll do that, then we know.